src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2464,16 +2464,16 @@ namespace NServiceBus.Sagas
     }
     public class SagaFinderDefinition
     {
-        public SagaFinderDefinition(System.Type type, string messageType, System.Collections.Generic.Dictionary<string, object> properties) { }
-        public string MessageType { get; }
+        public System.Type MessageType { get; }
+        public string MessageTypeName { get; }
```

Comment:
> the class is intended to be used (and currently is in all test that dogfood) in combination with either SagaMetadataCollection or SagaMetdata. 

Agreed. Both classes we're talking about are simple data containers. There's no logic. Unit testing them independently would be like unit testing a dropdown list item outside of the dropdown list.